ebean-test/src/test/resources/migrationtest/dbmigration/yugabyte/1.1.sql

```diff
@@ -55,13 +56,21 @@ alter table migtest_e_basic alter column status2 drop not null;
 alter table migtest_e_basic add constraint uq_migtest_e_basic_description unique  (description);
 
 insert into migtest_e_user (id) select distinct user_id from migtest_e_basic;
+commit transaction /* 3 */;
```

Comment:
@rbygrave what is the reason, that the duplicate lines are skipped in DdlRunner?

Reply:
Not sure.  What is the code that does that?

Reply:
Hmmm. No git history there, it has been moved from ddl-generator I believe.  Now I'd say that isn't necessary and we can remove that `noDuplicates` list altogether.